static/js/apps/nl_interface/debug_info.tsx

```diff
@@ -166,6 +167,11 @@ export function DebugInfo(props: DebugInfoProps): JSX.Element {
               Temporal classification: {debugInfo.temporalClassification}
             </Col>
           </Row>
+          <Row>
+            <Col>
+              TimeDelta classification: {debugInfo.timeDeltaClassification}
+            </Col>
+          </Row>
```

Comment:
While here, could you add debug info for comparison classification too please?

Reply:
Fine to defer to future PR.

Reply:
Defering to future PR